buildSrc/src/main/java/net/neoforged/neodev/CreateUserDevConfig.java

```diff
@@ -150,7 +155,11 @@ record UserDevConfig(
         List<String> libraries,
         List<String> testLibraries,
         Map<String, UserDevRunType> runs,
-        List<String> modules) {}
+        List<String> modules,
+        UserDevFeatures features) {}
+
+record UserDevFeatures(
+    boolean usesClasspathForRuntimeDetection) {}
```

Comment:
We need to find a better name. It's more something like `boolean noLegacyClasspath`?